src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -1657,6 +1657,8 @@ public static LivingEntity getLivingEntity(ModElement modElement, Random random,
 					new ProfessionEntry(modElement.getWorkspace(), getRandomDataListEntry(random, professions)));
 			livingEntity.tradeProfessionsExclude = !_true;
 		}
+		livingEntity.tradeAmbientSound = new Sound(modElement.getWorkspace(),

```

Comment:
I would just use livingSound for ambient sound, not sure if this parameter is also needed

Reply:
Or do you think there are cases where these two make sense to be different? If it is too specific of a use case, maybe better to not offer too many options in general

Reply:
I based myself on the WanderingTrader code. Its `getAmbientSound()` method returns a different sound depending on whether it is trading or not. Hence why I added a new parameter `tradeAmbientSound` so if it and basic ambient sound are given, they are used in the code.